### PR TITLE
Add a script to produce eval golden data for explore landing page queries

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: tools/nl/eval/fetch_explore_queries.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    return []
+  debug = response['debug']['debug']
+
+  single_sv_best_score = debug['sv_matching']['CosineScore'][0]
+  use_single_sv = True
+  multi_sv_candidate = None
+  if 'MultiSV' in debug['sv_matching']:
+    for candidate in debug['sv_matching']['MultiSV']['Candidates']:
+      if candidate['DelimBased'] and len(candidate['Parts']) == 2:
+        if candidate['AggCosineScore'] + 0.05 > single_sv_best_score:
```
Comment (chejennifer) [inline tools/nl/eval/fetch_explore_queries.py:41]: curious what the `+ 0.05` is for, maybe leave a comment?

File: tools/nl/eval/fetch_explore_queries.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+          query = example['title']
+          print(query)
+          dc_response = call_dc(query)
+          for item in dc_response:
+            for sv in item['sv']:
+              records.append({
+                  'query': item['query'],
+                  'used_query': item['used_query'],
+                  'sv': sv,
+                  'sentences': item['sv_to_sentences'].get(sv, ''),
```
Comment (chejennifer) [inline tools/nl/eval/fetch_explore_queries.py:88]: default should be `[]`?

**Review by chejennifer — COMMENTED** [review]
nice!

File: tools/nl/eval/fetch_explore_queries.py (hunk 1 above)
Comment (shifucun) [reply tools/nl/eval/fetch_explore_queries.py:43]: This is to match nl logic. Added a comment.

**Review by shifucun — COMMENTED** [review]
thanks for review!

**Review by chejennifer — APPROVED** [review]
thanks for the changes

**Pull request merged**